types/tape/package.json

```diff
@@ -2,12 +2,19 @@
     "private": true,
     "name": "@types/tape",
     "version": "5.6.9999",
+    "exports": {
+		".": "./index.js",
+		"./lib/default_stream": "./lib/default_stream.js",
+		"./lib/results": "./lib/results.js",
+		"./lib/test": "./lib/test.js",
```

Comment:
Do these files exist on DT? I don't see them, so I assume this will break?

Reply:
oh, hm - no, they don't - i just copied the exports field verbatim. should these be pointing to the d.ts files instead?

Reply:
Copying verbatim is fine, but it was more about the other `./lib/*` files seemingly not existing on DT.